packages/api/test/e2e/queries.e2e.ts

```diff
@@ -34,16 +32,12 @@ describe('e2e queries', () => {
     api = await initApiPromise();
   });
 
-  afterAll(async done => {
-    if (api) {
-      return await api.disconnect();
-    }
-    api = null;
-    done();
+  afterAll(async () => {
```

Comment:
this was always forcing a 60s timeout before 😅

Reply:
What is the timeout right now?

Reply:
60s configured in jest.init.js